src/control/vendor/github.com/daos-stack/go-spdk/spdk/src/nvme_control.c

```diff
@@ -101,8 +101,8 @@ attach_cb(void *cb_ctx, const struct spdk_nvme_transport_id *trid,
 		exit(1);
 	}
 
+	spdk_pci_addr_parse(&entry->pci_addr, trid->traddr);
```

Comment:
It's better to check return code.

Reply:
addressed in daos-stack/go-spdk#7 will be pulled in next time vendor version is bumped in DAOS